gradle.properties

```diff
@@ -0,0 +1 @@
+android.builder.sdkDownload=true
```

Comment:
It's what replaces Jake's SDK downloader. It's the official Google supported way to do it now.